cmd/consul-dataplane/main.go

```diff
@@ -107,7 +108,7 @@ func init() {
 
 	flag.BoolVar(&useCentralTelemetryConfig, "telemetry-use-central-config", true, "Controls whether the proxy applies the central telemetry configuration.")
 
-	flag.StringVar(&promRetentionTime, "telemetry-prom-retention-time", "", "The duration for Prometheus metrics aggregation.")
```

Comment:
I chose a default of 60s because that is the default go-metrics uses: https://github.com/hashicorp/go-metrics/blob/master/prometheus/prometheus.go#L22 🤷